R/tree_accessors.R

```diff
@@ -2877,33 +2877,83 @@ setMethod("cell_footnotes<-", "ContentRow",
   definition = .cfn_set_helper
 )
 
+# Deprecated methods
 #' @export
 #' @rdname ref_fnotes
 setGeneric("col_fnotes_here", function(obj) standardGeneric("col_fnotes_here"))
 #' @export
+#' @rdname ref_fnotes
+setMethod("col_fnotes_here", "ANY", function(obj) {
+  .Deprecated(
+    new = "col_footnotes",
+    old = "col_fnotes_here",
+    msg = "col_fnotes_here was deprecated in {rtables} version 0.6.5.9011. Please use col_footnotes instead."
+  )
+  col_footnotes(obj)
+})
+#' @export
+#' @rdname ref_fnotes
+setGeneric("col_fnotes_here<-", function(obj, value) standardGeneric("col_fnotes_here<-"))
+#' @export
 #' @rdname int_methods
-setMethod("col_fnotes_here", c("LayoutColTree"), function(obj) obj@col_footnotes)
+setMethod("col_fnotes_here<-", "ANY", function(obj, value) {
+  .Deprecated(
+    new = "col_footnotes<-",
+    old = "col_fnotes_here<-",
+    msg = "col_fnotes_here<- was deprecated in {rtables} version 0.6.5.9011. Please use col_footnotes<- instead."
+  )
+  col_footnotes(obj) <- value
```

Comment:
only a silly question: why dont we need here to have obj at the end while in other places we do?

Reply:
I'm not sure, possibly accidental? I don't think these functions were ever fully fleshed out